### PR TITLE
Add site introduction 

### PR DESCRIPTION
**Pull request by jonhartzler**
Beehive needs to have a separation between the site introduction and the introduction for the exhibit.  

To do this I added a field to the exhibit table, added the field to the exhibit form, and added the field to the api.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -44,6 +44,7 @@
     t.string   "image_content_type", limit: 255
     t.integer  "image_file_size",    limit: 4
     t.datetime "image_updated_at"
+    t.text     "short_description",  limit: 65535
```
Comment (danthewolfe) [inline db/schema.rb]: Is this too big of a limit?

Comment (jaronkk) [inline db/schema.rb]: This is the default limit for a text field.  Rails has started dumping the full limits in the schema.rb.  The only reason I might limit it is if MySQL has to allocate that much space per row, even if the full value isn't used (but I don't know exactly how it stores text)

Comment (jonhartzler) [inline db/schema.rb]: In general it seems every time I try and be judicious about the length all I ever get is an opportunity to extend it later.  I would only look at this if it was slow in the query.   That said it is probably too long.

Comment (danthewolfe) [inline db/schema.rb]: I kind of figured it was a default value. Can or should we limit the available text length in Redactor? (This is probably an issue that can be handled separately from this PR).

**Comment by danthewolfe** [comment]
:+1:

**Comment by jaronkk** [comment]
:+1:

**Pull request merged**